.travis.yml

```diff
@@ -2,7 +2,12 @@ language: scala
 jdk:
   - oraclejdk8
 scala:
-  - 2.11.6
+  - 2.11.7
+
 
 install: ./gradlew assemble -x signArchives
-script: ./gradlew  check -x signArchives
+script:
+  - jdk_switcher use openjdk6
+  - ./gradlew check -x signArchives -x java-it-tests:common:check -x java-it-tests:driver20:check -x java-it-tests:driver21:check
+  - jdk_switcher use oraclejdk8
+  - ./gradlew java-it-tests:common:check java-it-tests:driver20:check java-it-tests:driver21:check -x signArchives
```

Comment:
Yeah, I am a fan!  Seems like it has an answer for everything :).  The only thing I couldn't wrap my head around was finding a way to omit a module and it's children completely when building with gradle.  For now i'm excluding the check phase for each module individually in jdk6 which seems to work (`-x java-it-tests:common:check -x java-it-tests:driver20:check -x java-it-tests:driver21:check`)